### ISSUE TITLE
Reinstate label localization using depth data

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2902]
##### Brief description of problem/feature
After months of having no depth data from Google, we removed our support for it in #2632 because it added a lot to code complexity. But the depth data endpoint has actually been back up for the last few months... I'm thinking that we should add this code back in so that we can have more accurate label lat/lngs for as long as this endpoint remains up?

**Comment by jonfroehlich** [comment]
Amazing. Is there an elegant way to add it back in (e.g., a separate .js file and a conditional that only uses depth triangulation if it exists)

**Comment by misaugstad** [comment]
Yep! I think we can keep it pretty short and sweet like that.

**Comment by misaugstad** [comment]
I'm not really sure what's going on... I went to bring the depth data back, but when I tried using our usual URLs to get depth data, they are returning 404 errors for me. Like the URL below should work:
https://maps.google.com/cbk?output=xml&cb_client=maps_sv&hl=en&dm=1&pm=1&ph=1&renderer=cubic,spherical&v=4&panoid=tOANCSOy2o9zE39RfURKeA

The reason I'm really confused is because this URL seems to be working on the pano scraper that is live and running right now. The XML with depth data is being downloaded successfully every night. But if I try to use the same URL the following day to get the same XML, it doesn't work!

I even tried running the pano scraper locally, and the downloads were failing through that as well! I have no idea why the downloads are working on the live pano scraper, but not the one that I spun up locally..? I might try running the scraper locally from my home wifi instead of Starbucks wifi at some point over the next week. But for now I'm just really confused! And can't move forward with this while I'm failing to download depth data 🤷
